linalg/sparsemat.hpp

```diff
@@ -424,7 +424,7 @@ class SparseMatrix : public AbstractSparseMatrix
    /// Returns whether or not CSR format has been finalized.
    bool Finalized() const { return !A.Empty(); }
    /// Returns whether or not the columns are sorted.
-   bool areColumnsSorted() const { return isSorted; }
+   bool ColumnsAreSorted() const { return isSorted; }
```

Comment:
@tzanio, please do not sneak in interface changes like this -- just because you didn't like the name, and right before merging in master.

Reply:
This was an editorial decision that I discussed with @jandrej in person. It was not a question of taste, but of conformity with the current naming practice in MFEM.